tools/prlint/test/module.test.ts

```diff
@@ -0,0 +1,48 @@
+import { findModulePath, moduleStability } from '../module';
+import * as path from 'path';
+
+const repoRoot = path.join(__dirname, '..', '..', '..');
+
+describe('findModulePath', () => {
+  test('single fuzzy match', () => {
+    expect(relative(findModulePath('lambda'))).toEqual('packages/@aws-cdk/aws-lambda');
+    expect(relative(findModulePath('s3'))).toEqual('packages/@aws-cdk/aws-s3');
+    expect(relative(findModulePath('cdk-build-tools'))).toEqual('tools/cdk-build-tools');
+  });
+
+  test('multiple fuzzy matches', () => {
+    // also matches 'packages/@monocdk-experiment/assert'
+    expect(relative(findModulePath('assert'))).toEqual('packages/@aws-cdk/assert');
+
+    // also matches 'packages/aws-cdk' and 'tools/eslint-plugin-cdk'
+    expect(relative(findModulePath('cdk'))).toEqual('packages/cdk');
+  });
+
+  test('no matches', () => {
+    expect(() => findModulePath('doesnotexist')).toThrow(/No module/);
+  });
+
+  function relative(loc: string) {
+    return path.relative(repoRoot, loc);
+  }
+});
+
+describe('moduleStability', () => {
+  test('happy', () => {
+    expect(moduleStability(absolute('packages/@aws-cdk/aws-lambda'))).toEqual('stable');
+    expect(moduleStability(absolute('packages/@aws-cdk/aws-s3'))).toEqual('stable');
+
+    expect(moduleStability(absolute('packages/@aws-cdk/aws-apigatewayv2'))).toEqual('experimental');
```

Comment:
this also feels flaky...I'm wondering if the tests could create an experimental module?

Reply:
I agree, this ties us not just to a specific module stability at a given time - which can change, it also ties us to the experimental modules being in the main repo and under `packages/@aws-cdk/`, both might change in the future.

I would just remove the tests for experimental modules, the important part is to verify that we will never allow a breaking change to a stable module. The test for experimental modules is nice to have but we can skip it if it introduce maintenance overhead or is just too hard to implement.

Reply:
Ok, removed these tests.